native/acter/api.rsh

```diff
@@ -592,6 +604,52 @@ object Rsvp {
     fn status() -> string;
 }
 
+//  ### ##   ### ###    ##      ## ##   #### ##    ####    ## ##   ###  ##   ## ##   
+//  ##  ##   ##  ##     ##    ##   ##  # ## ##     ##    ##   ##    ## ##  ##   ##  
+//  ##  ##   ##       ## ##   ##         ##        ##    ##   ##   # ## #  ####     
+//  ## ##    ## ##    ##  ##  ##         ##        ##    ##   ##   ## ##    #####   
+//  ## ##    ##       ## ###  ##         ##        ##    ##   ##   ##  ##      ###  
+//  ##  ##   ##  ##   ##  ##  ##   ##    ##        ##    ##   ##   ##  ##  ##   ##  
+// #### ##  ### ###  ###  ##   ## ##    ####      ####    ## ##   ###  ##   ## ## 
+
+
+object ReactionManager {
+
+    /// get my reaction status
+    fn my_status() -> Future<Result<bool>>;
+
+    /// whether manager has reaction entries
+    fn has_reaction_entries() -> bool;
+
+    /// get total count of reactions
+    fn total_reaction_count() -> u32;
+
+    /// get reaction entries
+    fn reaction_entries() -> Future<Result<Vec<Reaction>>>;
+
+    /// send the reaction
+    fn send_reaction(event_id: string, key: string) -> Future<Result<EventId>>;
+
+    /// redact the reaction
+    fn redact_reaction(event_id: string, reason: Option<string>, txn_id: Option<string>) -> Future<Result<EventId>>;
```

Comment:
why do we have `event_id`s here? These should be know to the manager, no?

Reply:
Yeah, it is possible to remove `event_id` from api param in `send_reaction()`.
We can use event id of news without passing of event id as fn param.
But for `redact_reaction()`, we have to use reacted event id, not news event id.
So we can't remove `event_id` from api param in `redact_reaction()`.